### DISCUSSION TITLE
Can't run the cpu benchmark.

### DISCUSSION THREAD

**Discussion by Kosterc1** [RainbowMiner/RainbowMiner#1738] [Q&A]
The cpu device causes my pc to completly freeze up. Including rainbowminer. Can we get a setting to only run 3 treads? I tried to just edit the config file, but it gets overruled by something.

**Comment by RainbowMiner** [comment]
Please tell me more:
- which kind of CPU is it?
- what exactly did you put into the CPUMining paramter fields?
- did you fill in both CPU mining related parameters?
e.g.
```
  "CPUMiningThreads": 2,
  "CPUMiningAffinity": "0x0005",
```

**Comment by Kosterc1** [comment]
The problem is solved, but needs work. What happened was, it auto started
benchmarking with 4 treads. Instant freeze of system. In the settings it
was 3. But 3 doesn't exist it seems. Had to use the line below. To free
1-2. So it only uses 2 treads. Not 3. But at least it doesn't freeze the
pc. But what bugged me most, I had to use c in the console. No way to kill
the cpu miner, to reduce the load. Local is not accessible if not mining.
Pausing or hitting c did nothing. So took a while to get this back on the
road.
